pkg/storage/db_test.go

```diff
@@ -1291,15 +1249,7 @@ func TestGetARNIDAtTimeMoreThanOneChampions(t *testing.T) {
 		"aws_region_region",
 		"aws_resource_meta",
 		"aws_resource_aws_account_id",
-	}).AddRow("172.16.3.3",
-		"44.33.22.11",
-		"yahoo.com",
-		"type",
-		"aid",
-		"region",
-		[]byte("{\"hi\":\"there3\"}"),
-		1)
-	rows2 := sqlmock.NewRows([]string{"aws_account_account",
+		"aws_account_account",
 		"owner_login",
 		"oener_email",
```

Comment:
Unrelated to your changes, but this looks like typo in column name. The test still works though, as we `Scan` not by name but by position

Reply:
I looked further in the test and it seems like we have been using naming convention tablename_field. Test refactor task could be a good candidate to address this or leave as is.

Reply:
I mean `oEner` instead of `owner` :)